plugins/org.opencast.usertracking.MatomoSaverPlugIn/usertracking_matomo_saver.js

```diff
@@ -84,28 +84,28 @@ paella.addPlugin(function() {
 
         switch (event) {
           case paella.events.play:
-            paella.userTracking.matomotracker.trackEvent("Player.Controls","Play");
+            paella.userTracking.matomotracker.trackEvent("Player.Controls","Play", paella.loaderFunctionParams.videoLoader.metadata.title);
```

Comment:
The correct way to get the title is something like 

https://github.com/polimediaupv/paella/blob/e8f7620427102ae17d2023e9701950d5d8b184ff/plugins/es.upv.paella.windowTitlePlugin/window_title.js#L17-L21

Please change it to be approved